src/shared/symbols.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+import Symbol from 'es-symbol'
+
+export const ACTION_HANDLER = Symbol('action creator handler')
```

Comment:
I hate to expose these because then you'd have a lot of access to alt's internals :confused: